ui/externs/ui.js

```diff
@@ -71,6 +71,8 @@ shaka.extern.UIVolumeBarColors;
  *   addSeekBar: boolean,
  *   addBigPlayButton: boolean,
  *   customContextMenu: boolean,
+ *   fastForwardOnTaps: boolean,
```

Comment:
I feel like these could be an optional number rather than a boolean, and the number describes how far they go with one tap.

So like:
`null` means the feature is disabled.
`5` means it jumps in intervals of 5s.
`10` means it jumps in intervals of 10s.
Etc etc.

You could pass this value in to the element during creation time, which should work well with the suggestion I made about the superclass.

It's relevant so that the developer can have the seeking intervals be different from each other. For example, a podcast app I use for iOS has the seeking intervals of -10s and +45s. I've also seen apps that seek forward a small amount but back a large amount.